cmake/submoduleOpenMesh.cmake

```diff
@@ -0,0 +1,55 @@
+set (CMAKE_CXX_FLAGS "-fexceptions")
```

Comment:
Here is a big issue: this line completely overrides the CMAKE_CXX_FLAGS for all Radium.

The thing here is that OpenMesh requires exceptions to be enabled when it is compiled.
However, since some exceptions management is done in hpp files (see OpenMesh/Core/IO/StoreRestore.hh), when compiling OpenMesh's integration within Radium, a "exception handling disabled" error is thrown.

Hence, we must decide on whether enabling exceptions OR not using OpenMesh (or finding some kind of a hack).